### PR TITLE
Fix my personal keymap // Custom keymap for Kbdfans/kbd67/rev2 with improvements on Accessibility

### PR DESCRIPTION
**Pull request by catrielmuller**
## Description

I needed create a keyboard for a person that can only use one finger at the time. So I changed the all mod keys to the One Keypress variation, so all key bindings can be done with one finger.

The `console = no` it's to reduce the size of the binary to support Unicode

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by catrielmuller** [comment]
> Can we rename the `kbd67/rev2:cami` keymap to `kbd67/rev2:catrielmuller` so that it matches your existing `dztech/dz65rgb:catrielmuller` keymap.

I changed on the last commit to `catrielmuller_camilad` because that keyboard was a gift to a person that have some motor problems in the hands, so this keymap it's for she can use the all keyboard with a single finger.
I will check on the weekend if i can implement on the makefile the possibility to handle subfolders for the keymaps like `catrielmuller/camilad`.
I don't wanna use only my username because i have other kbd67, so probably i will create another keymap for the other board

**Comment by zvecr** [comment]
Ah thats a valid use case, in which case i think would accept either what you had or what is there now. I had assumed that it was a personal keymap... you know what they say about assumptions. 

As for nested keymap folders, I don't think that's something we would want to support right now.

**Comment by catrielmuller** [comment]
> Ah thats a valid use case, in which case i think would accept either what you had or what is there now. I had assumed that it was a personal keymap... you know what they say about assumptions.
> 
> As for nested keymap folders, I don't think that's something we would want to support right now.

Ok, so i think with a lodash it's a good way to differentiate that.
Lmk if you have another comment of this PR. Thanks for your time.

File: keyboards/kbdfans/kbd67/rev2/keymaps/catrielmuller_camilad/keymap.c [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  KC_CAPS      ,KC_A         ,KC_S         ,KC_D   ,KC_F   ,KC_G   ,KC_H   ,KC_J   ,KC_K   ,KC_L         ,KC_SCLN,KC_QUOT      ,KC_ENT         ,KC_PGDN, \
+  OSM(MOD_LSFT),KC_Z         ,KC_X         ,KC_C   ,KC_V   ,KC_B   ,KC_N   ,KC_M   ,KC_COMM,KC_DOT       ,KC_SLSH,OSM(MOD_RSFT)        ,KC_UP  ,KC_END , \
+  OSM(MOD_LCTL),OSM(MOD_LGUI),OSM(MOD_LALT),KC_SPC                                         ,OSM(MOD_RALT),OSL(1) ,OSM(MOD_RCTL),KC_LEFT,KC_DOWN,KC_RGHT),
+
+[1] = LAYOUT_65_ansi(
+   KC_GRV ,KC_F1  ,KC_F2  ,KC_F3  ,KC_F4  ,KC_F5  ,KC_F6  ,KC_F7  ,KC_F8  ,KC_F9  ,KC_F10 ,KC_F11 ,KC_F12 ,KC_DEL ,KC_INS , \
+   _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,KC_PSCR, \
+   _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,TILDE  ,_______        ,_______, \
+   _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______        ,KC_PGUP,_______, \
+   _______,_______,_______,_______                                                ,_______,_______,_______,KC_HOME,KC_PGDN,KC_END ),
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/rev2/keymaps/catrielmuller_camilad/keymap.c:35]:
```suggestion
  KC_ESC       ,KC_1         ,KC_2         ,KC_3   ,KC_4   ,KC_5   ,KC_6   ,KC_7   ,KC_8   ,KC_9         ,KC_0   ,KC_MINS      ,KC_EQL ,KC_BSPC,KC_HOME,
  KC_TAB       ,KC_Q         ,KC_W         ,KC_E   ,KC_R   ,KC_T   ,KC_Y   ,KC_U   ,KC_I   ,KC_O         ,KC_P   ,KC_LBRC      ,KC_RBRC,KC_BSLS,KC_PGUP,
  KC_CAPS      ,KC_A         ,KC_S         ,KC_D   ,KC_F   ,KC_G   ,KC_H   ,KC_J   ,KC_K   ,KC_L         ,KC_SCLN,KC_QUOT      ,KC_ENT         ,KC_PGDN,
  OSM(MOD_LSFT),KC_Z         ,KC_X         ,KC_C   ,KC_V   ,KC_B   ,KC_N   ,KC_M   ,KC_COMM,KC_DOT       ,KC_SLSH,OSM(MOD_RSFT)        ,KC_UP  ,KC_END ,
  OSM(MOD_LCTL),OSM(MOD_LGUI),OSM(MOD_LALT),KC_SPC                                         ,OSM(MOD_RALT),OSL(1) ,OSM(MOD_RCTL),KC_LEFT,KC_DOWN,KC_RGHT),

[1] = LAYOUT_65_ansi(
   KC_GRV ,KC_F1  ,KC_F2  ,KC_F3  ,KC_F4  ,KC_F5  ,KC_F6  ,KC_F7  ,KC_F8  ,KC_F9  ,KC_F10 ,KC_F11 ,KC_F12 ,KC_DEL ,KC_INS ,
   _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,KC_PSCR,
   _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,TILDE  ,_______        ,_______,
   _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______        ,KC_PGUP,_______,
   _______,_______,_______,_______                                                ,_______,_______,_______,KC_HOME,KC_PGDN,KC_END ),
```

File: keyboards/kbdfans/kbd67/rev2/keymaps/catrielmuller_camilad/keymap.c [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  KC_ESC       ,KC_1         ,KC_2         ,KC_3   ,KC_4   ,KC_5   ,KC_6   ,KC_7   ,KC_8   ,KC_9         ,KC_0   ,KC_MINS      ,KC_EQL ,KC_BSPC,KC_HOME, \
+  KC_TAB       ,KC_Q         ,KC_W         ,KC_E   ,KC_R   ,KC_T   ,KC_Y   ,KC_U   ,KC_I   ,KC_O         ,KC_P   ,KC_LBRC      ,KC_RBRC,KC_BSLS,KC_PGUP, \
+  KC_CAPS      ,KC_A         ,KC_S         ,KC_D   ,KC_F   ,KC_G   ,KC_H   ,KC_J   ,KC_K   ,KC_L         ,KC_SCLN,KC_QUOT      ,KC_ENT         ,KC_PGDN, \
+  OSM(MOD_LSFT),KC_Z         ,KC_X         ,KC_C   ,KC_V   ,KC_B   ,KC_N   ,KC_M   ,KC_COMM,KC_DOT       ,KC_SLSH,OSM(MOD_RSFT)        ,KC_UP  ,KC_END , \
+  OSM(MOD_LCTL),OSM(MOD_LGUI),OSM(MOD_LALT),KC_SPC                                         ,OSM(MOD_RALT),OSL(1) ,OSM(MOD_RCTL),KC_LEFT,KC_DOWN,KC_RGHT),
+
+[1] = LAYOUT_65_ansi(
+   KC_GRV ,KC_F1  ,KC_F2  ,KC_F3  ,KC_F4  ,KC_F5  ,KC_F6  ,KC_F7  ,KC_F8  ,KC_F9  ,KC_F10 ,KC_F11 ,KC_F12 ,KC_DEL ,KC_INS , \
+   _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,KC_PSCR, \
+   _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,TILDE  ,_______        ,_______, \
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/rev2/keymaps/catrielmuller_camilad/keymap.c:33]:
```suggestion
   _______,_______,_______,_______,_______,_______,_______,_______,_______,_______,_______,KC_TILD,_______        ,_______, \
```

File: keyboards/kbdfans/kbd67/rev2/keymaps/catrielmuller_camilad/keymap.c [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case TILDE:
+      if (record->event.pressed) {
+        SEND_STRING("~");
+      }
+      break;
+  }
+  return true;
+}
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/rev2/keymaps/catrielmuller_camilad/keymap.c:48]:
```suggestion
```

File: keyboards/kbdfans/kbd67/rev2/keymaps/catrielmuller_camilad/keymap.c [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum custom_keycodes {
+  TILDE = SAFE_RANGE
+};
+
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/rev2/keymaps/catrielmuller_camilad/keymap.c:20]:
```suggestion
```

**Comment by catrielmuller** [comment]
Thanks @fauxpark

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**